docs/conf/na/2017/writing-day.rst

```diff
@@ -28,3 +28,52 @@ Documenting a new project?
 Check out our `beginners guide <http://docs.writethedocs.org/writing/beginners-guide-to-docs/>`_ to writing documentation.
 This should help you get started,
 and give you some ideas for how you can contribute to a project that you love.
+
+Write Write the Docs on Writing Day
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This year we're also running a session where you can help improve your favourite website. Yup, you can brainstorm improvements, write helpful content and posts or just magically improve `Write the Docs <http://www.writethedocs.org>`_.
+
+Ideally you'll already be familiar with GitHub and writing in plain text (markdown or restructured text), but we'll be there to help out if you're not. And if you'd rather write in Google Docs or program in python or jinja we've got you covered.
+
```

Comment:
"Markdown or reStructuredText ..."
"program in Python or Jinja ..."
In fact, this might be better rewritten altogether to welcome other sorts of contribution. (I'm thinking especially information architecture, which doesn't necessarily require any markup skilz.)
Say:
"If you want to edit the site directly, be aware that we use Markdown and reStructuredText. And our source files live on GitHub. If you're familiar with these tools already, great. If not, you can learn, or you can contribute in other ways, for example by helping reorganize content or redesign information architecture.